prov/gni/src/gnix_tags.c

```diff
@@ -930,6 +930,7 @@ void _gnix_remove_tag(
 		struct gnix_tag_storage *ts,
 		struct gnix_fab_req *req)
 {
+	GNIX_TRACE(FI_LOG_EP_DATA, "\n");
```

Comment:
i'm concerned this may be adding overhead in the tag match engine.  Could we change this to GNIX_DEBUG?